docs/dev/awips-development-environment.md

```diff
@@ -57,69 +63,102 @@ If it's not already installed, install git:
+## 5. Configure Eclipse
 
-Open eclipse by running: `/awips2/eclipse/eclipse.sh`
+Open eclipse by running: `/awips2/eclipse/eclipse`
 
-Verify or make the following changes to set up eclipse for AWIPS development:
+It is fine to choose the default workspace upon starting up.
 
-* Preferences > Java 
-    
-    Set to **/awips2/java**
+### Set Preferences
 
-* Preferences > PyDev > Python Interpreter
+Verify or make the following changes to set up eclipse for AWIPS development:
 
-    Set to **/awips2/python/bin/python**
+1. Window > Preferences > Java > Installed JREs
     
-* There might be some unresolved errors.  These should be made to warnings instead.
+     * Set to **/awips2/java**
+
+1. Window > Preferences > PyDev > Interpreters > Python Interpreter
 
-    Preferences > Java > Compiler > Building > **Circular Dependencies** > Change to Warning
-    Preferences > Plug-in Development > API Baselines > **Missing API Baseline** > Change to Warning
+      * Set to **/awips2/python/bin/python**
     
-* **Turn off automatic building** (you will turn this back on after importing the repos)
+        > Note: Add all paths to the SYSTEM pythonpath if prompted
     
-    Project > Uncheck "Build Automatically"
+1. There might be some unresolved errors.  These should be made to warnings instead.
 
-* File > Import > General > Existing Projects Into Workspace
-
-    Import all of the git cloned project folders **EXCEPT** for the main (first) **github.com/Unidata/awips2.git** directory (which should be **~/awips2**).  
-    Select **awips2-core**, **awips2-core-foss**, **awips2-foss**, **awips2-ncep**, etc. > Select All Projects > Finish 
-     
-    You'll want to import **~/awips2** in two parts to ensure a clean and error-free Eclipse build:
+      * Window > Preferences > Java > Compiler > Building > Build path Problems > **Circular Dependencies** > Change to Warning
+      * Window > Preferences > Plug-in Development > API Baselines > **Missing API Baseline** > Change to Warning
     
-    1. Import **awips2/cave** > Select All Projects > Finish
-    2. Import **awips2/edexOsgi** > Select All Projects > Finish
-
-* Project > Clean
-
-    Clean the build and ensure no errors are reported.  
+1. **Turn off automatic building** (you will turn this back on after importing the repos)
     
-* Turn automatic building back on
+      * Project > Uncheck "Build Automatically"
+
+### Importing Git Repos
+
+All of the git repos that were [cloned in the previous step](#4-download-the-source-code) will need to be imported into Eclipse.  **But, be aware the `awips2` repo is done last,** because it requires different steps.
+
+1. File > Import > Git > Projects from Git > **Next**
+  ![git import](../images/gitImport1.png)
+  
+1. Continue with the default selection, Existing local repository > **Add..** > add each of the git repos (for example `.../awips2-core`) > check the checkbox > **Finish**
+  ![add git repo](../images/gitImport2.png)
```

Comment:
Your path in the image is /home/awips/awipsRepos/**_unusedRepos_**/awips2-core

Might change that path